sharding/nodesSetup.go

```diff
@@ -241,52 +170,39 @@ func (ns *NodesSetup) processConfig() error {
 		ns.nrOfNodes++
 	}
 
-	for _, consensusConfig := range ns.chainParametersByEpoch {
-		if consensusConfig.ShardConsensusGroupSize < 1 {
-			return ErrNegativeOrZeroConsensusGroupSize
-		}
-		if consensusConfig.ShardMinNumNodes < consensusConfig.ShardConsensusGroupSize {
-			return ErrMinNodesPerShardSmallerThanConsensusSize
-		}
-		if ns.nrOfNodes < consensusConfig.ShardMinNumNodes {
-			return ErrNodesSizeSmallerThanMinNoOfNodes
-		}
-		if consensusConfig.MetachainMinNumNodes < 1 {
-			return ErrNegativeOrZeroConsensusGroupSize
-		}
-		if consensusConfig.MetachainMinNumNodes < consensusConfig.MetachainConsensusGroupSize {
-			return ErrMinNodesPerShardSmallerThanConsensusSize
-		}
+	if ns.genesisChainParameters.ShardConsensusGroupSize < 1 {
```

Comment:
wondering if it is better to move some of these checks in the chainParametersHolder